modules/node/src/transfer/transfer.service.ts

```diff
@@ -216,10 +216,14 @@ export class TransferService {
           "DEPOSIT_CONFIRMED_EVENT",
           async (msg: DepositConfirmationMessage) => {
             if (msg.from !== this.cfCoreService.cfCore.publicIdentifier) {
-              return;
+              reject(
+                `Deposit event from field: ${msg.from}, did not match public identifier: ${this.cfCoreService.cfCore.publicIdentifier}`,
+              );
             }
             if (msg.data.multisigAddress !== channel.multisigAddress) {
-              return;
+              reject(
+                `Deposit event multisigAddress: ${msg.data.multisigAddress}, did not match channel multisig address: ${channel.multisigAddress}`,
+              );
```

Comment:
again, seems a little weird to reject here since it  could be an irrelevant event thrown.

theres no error handling for these rejections so the payment would just fail